pytket/qir/conversion/api.py

```diff
@@ -58,6 +60,7 @@ def pytket_to_qir(
     :param int_type: size of each integer, allowed value 32 and 64
     :param cut_pytket_register: breaks up the internal scratch bit registers
       into smaller registers, default value false
+    :param profile: generates qir corresponding to the standart profile
```

Comment:
```suggestion
    :param profile: generates QIR corresponding to the standard profile
```
What is the "standard profile"? Is there a reference for this? And what profile is generated if the flag is not set? I think some more explanation is needed.

Reply:
I will add some more details.

Reply:
The new adaptive profile is not yet online, I have added two links in [904c87f](https://github.com/CQCL/pytket-qir/pull/163/commits/904c87faa156d11d43d83eb611123798d1ece4a2)